routes/index.mjs

```diff
@@ -1,9 +1,30 @@
 import express from 'express';
 const router = express.Router();
+import { check, validationResult } from 'express-validator';
 
 /* GET home page. */
 router.get('/', (req, res, next) => {
-  res.render('main/index', { title: 'Express' });
+  res.render('main/index', {csrfToken: req.csrfToken(), errors: {}, data: {} });
+});
+
+// This is a basic example of express validation and should not be viewed as a development standard.
+router.post('/', [
+  check('email').isEmail().withMessage('Invalid email').normalizeEmail(),
+  check('fullName').notEmpty().withMessage('Name is required').trim().escape(),
+  check('subscribe', 'Please select an option').notEmpty()
+], (req, res) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.render('main/index', {
+      csrfToken: req.csrfToken(),
+      errors: errors.mapped(),
+      data: req.body
+    });
+  }
+  // on success
+  res.render('main/index', {
+    data: {success: true}
+  });
```

Comment:
Question - is this rationale of having this in the `index.mjs` as opposed to the `README.md`,  just so we can see a working example on the front page? Would we want to add some bits of this to the `README.md`?

Reply:
It's worth noting in the README.md but that's more for the ticket you're doing at the moment. 

We wanted a form on the index page so that there is at least one working example. We can take it out on a later date when we have a fully working demo using the skeleton. Security need at least something to use in order to help access the projects security from an API and Frontend prospective.

Reply:
Good point, I'll make sure to add this in.